### PR TITLE
Bump MVDTool/PyMvdTool version to 2.3.6

### PR DESCRIPTION
**Pull request by WeinaJi**

### CODE REVIEW

File: var/spack/repos/builtin/packages/mvdtool/package.py [hunk 1]
```diff
@@ -16,6 +16,7 @@ class Mvdtool(CMakePackage):
     git      = "https://github.com/BlueBrain/MVDTool.git"
 
     version('develop', get_full_repo=False, submodules=True)
+    version('2.3.6', tag='v2.3.6', get_full_repo=False)
```
Comment (WeinaJi) [inline var/spack/repos/builtin/packages/mvdtool/package.py:19]: @matz-e  Can we remove some of the old versions below from the recipe ? Which are still needed ?

Comment (matz-e) [reply var/spack/repos/builtin/packages/mvdtool/package.py:19]: I guess we could, there's always the possibility to go back in Spack history. Is there any harm leaving them in, though?

**Review by matz-e — APPROVED** [review]

File: var/spack/repos/builtin/packages/mvdtool/package.py (hunk 1 above)
Comment (WeinaJi) [reply var/spack/repos/builtin/packages/mvdtool/package.py:19]: No harm, I am just wondering because the list is getting long. We can also keep them in.

Comment (matz-e) [reply var/spack/repos/builtin/packages/mvdtool/package.py:19]: Maybe only the latest `2.3`, `2.2` etc version can be kept? I'll leave it up to you if you want to remove them

**Pull request merged**